config/src/vespa/config/common/configparser.h

```diff
@@ -14,81 +14,95 @@ namespace config {
  */
 class ConfigParser {
 public:
-    using vsvector = StringVector;
+    class Cfg {
+    public:
+        Cfg(const std::vector<vespalib::string> & v)
```

Comment:
Explicit implicit to get automatic conversion. Intention to make another pass at this to avoid the vector of strings completely in not too distant future.